dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1021,6 +1029,148 @@ def _get_percentile(self, percentiles):
             quantiles[percentiles == 50] = median_value
         return quantiles.tolist()
 
+    @property
+    def median_absolute_deviation(self):
```

Comment:
fix name to matchs scipy

Reply:
which existing? I made it consistent across the file and the option.

Reply:
changed to median_abs_deviation